packages/base/cards-grid.gts

```diff
@@ -303,6 +303,7 @@ class Isolated extends Component<typeof CardsGrid> {
         cursor: pointer;
         container-name: fitted-card;
         container-type: size;
+        padding: 1px;
```

Comment:
Seems like this might be at odds with how we expected container queries to work. @burieberry can you advise?

Reply:
I think this is because we're using `box-shadow` in the `CardContainer` to create borders instead of the `border` property. Since overflow is hidden, the box-shadow is not showing up unless we give the 1px padding. The reason for using box-shadow was because the switch to selection highlight looks more elegant cleanly replacing it without layout shift. However, the 1px padding caused it to not look so clean because now we see the faint outline of the card border even when it's highlighted:
<img width="420" alt="highlighted" src="https://github.com/user-attachments/assets/9d8d99bf-4cfc-4663-ae69-f91d0852dc66">

I recommend putting the `overflow: hidden` property on the `.fitted-template` class and not on the `.card` class. (I actually noticed text-overflow ellipses not working in some cases because of this being missing). The cards-grid `ul` container will probably need a 1px padding to the left but that's better than making people put 1px padding on the cards themselves because we can offer the cards-grid component with this built-in.

Another thing, unrelated: The `cursor: pointer` property should be on the interactive element (card overlay) and not on the card itself.

Reply:
@FadhlanR please create a follow up PR to address @burieberry's comment